### ISSUE TITLE
Manual Install instructions fail to create ui/build directory

### ISSUE THREAD

**Issue by myuseraccount** [photoview/photoview#811]
**Describe the bug**
I've tried following the manual install instructions, but the step to build the UI does not create the directory as described.  It doesn't matter if I download the tarball, or do a git clone of the repository.

**To Reproduce**
1. Follow the manual install instructions at [photoview.github.io](https://photoview.github.io/en/docs/installation-manual/)
2. After the step [Build the web user-interface](https://photoview.github.io/en/docs/installation-manual/#build-the-web-user-interface), there is no ui/build directory.
3. Continuing to the step [Copy needed files](https://photoview.github.io/en/docs/installation-manual/#copy-needed-files) fails when instructed to copy files from ui/build.

**Expected behavior**
The instructions should be able to be followed as written, so that they produce and use the correct folder to build the UI and install it.

**Your environment:**
I am trying to build on Ubuntu 22.04.  I updated to current versions of Go and NPM.

 - Are you using Docker?  No
 - Database [haven't gotten that far]
 - Browser [N/A]
 - Version [2.3.13]

Sorry if someone else has already submitted this, but I can't find an existing bug report for it.

**Comment by myuseraccount** [comment]
I think I may have found it: by wiping and starting over, it looks like the directory created by this step is called "dist".

When I continue on, though, I get an error building the API, saying that a file needed by a dependency is missing:

> github.com/strukturag/libheif/go/heif
> go build github.com/strukturag/libheif/go/heif:
> # pkg-config --cflags  -- libheif
> Package dav1d was not found in the pkg-config search path.
> Perhaps you should add the directory containing `dav1d.pc'
> to the PKG_CONFIG_PATH environment variable
> Package 'dav1d', required by 'libheif', not found
> pkg-config: exit status 1

**Comment by viktorstrate** [comment]
Did you install the needed dependencies under the preparation step in the guide?
Specifically the `libheif` package

**Comment by myuseraccount** [comment]
I installed libheif-dev as instructed, in the long apt install command. The installation doesn't say to install libde265 other than to install the PPA for each.  But perhaps it was installed as a dependency of libde265, because apt tells me libheif1 and libheif-dev are already installed.  

```
$ dpkg -L libheif-dev | more
/.
/usr
/usr/include
/usr/include/x86_64-linux-gnu
/usr/include/x86_64-linux-gnu/libheif
/usr/include/x86_64-linux-gnu/libheif/heif.h
/usr/include/x86_64-linux-gnu/libheif/heif_cxx.h
/usr/include/x86_64-linux-gnu/libheif/heif_plugin.h
/usr/include/x86_64-linux-gnu/libheif/heif_version.h
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/pkgconfig
/usr/lib/x86_64-linux-gnu/pkgconfig/libheif.pc
/usr/share
/usr/share/doc
/usr/share/doc/libheif-dev
/usr/share/doc/libheif-dev/README.md.gz
/usr/share/doc/libheif-dev/changelog.Debian.gz
/usr/share/doc/libheif-dev/copyright
/usr/lib/x86_64-linux-gnu/libheif.so

```
So, how is dav1d supposed to be installed?

Back to the original issue: does ui/dist contain what the instructions refer to as ui/build?

**Comment by viktorstrate** [comment]
Yes, `ui/build` might have been renamed to `ui/dist`

**Comment by myuseraccount** [comment]
Builds seem to work now, and I replaced `ui/build` with `ui/dist` while following the instructions.  Looks like it needs to be updated in the systemd install section as well.

Speaking of which, now it's exiting when I try to start it with systemctl, but that's something for another issue.

**Comment by myuseraccount** [comment]
I figured out a few more non-obvious things that could also be helped by updating the manual install instructions.

In the systemd section, it tells you two things to omit if you aren't using SQLite: the second line of `systemd/photoview.tempfiles`, and the command `sudo install -d "/var/lib/photoview"`.  However, there's a line in `systemd/photoview.service` that also references `/var/lib/photoview`, so if that isn't present, it still fails.  I ended up just creating that directory and that allowed it to run, before I figured out there was a reference to it in the `photoview.service` file.  I haven't tried removing that line and `/var/lib/photoview` to confirm that it works.

Also, not everything needed in `/etc/systemd.env` is explained under manual install, but the page on environment variables had some clues.  I had to set `PHOTOVIEW_MEDIA_CACHE=/var/cache/photoview/media_cache/` and `PHOTOVIEW_SERVE_UI=1`, but `PHOTOVIEW_PUBLIC_ENDPONT` was unnecessary.  

I'm guessing I'm an outlier, trying to install on the bare OS with systemd, and most other folks just use Docker, or someone would have noticed these a while ago.

Darktable gets an error in the logs, so I'm going to file another issue for that, but I'm glad to have the server running finally!
